src/Pages/ProductListPage/ProductListComponent/Products/Products.jsx

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import './Products.scss';
+import ProductsInfo from './ProductsInfo/ProductsInfo';
+import ProductsItem from './ProductsItem/ProductsItem';
+import MovePage from './MovePage/MovePage';
+class Products extends Component {
+  render() {
+    console.log(this.props.prList);
+    return (
+      <main className="Products">
```

Comment:
```suggestion
import React, { Component } from 'react';
import ProductsInfo from './ProductsInfo/ProductsInfo';
import ProductsItem from './ProductsItem/ProductsItem';
import MovePage from './MovePage/MovePage';
import './Products.scss';

class Products extends Component {
  render() {
    return (
      <main className="Products">
```
- import 순서 지켜주세요. (React → Library(Package) → Component → 변수 / 이미지 → css 파일(scss 파일))
- 구분되는 요소들끼리는 한줄 씩 띄워서 가독성 확보해주세요
- 테스트 끝났으면 console.log 삭제해주세요